SimPEG/EM/NSEM/SurveyNSEM.py

```diff
@@ -1,3 +1,14 @@
+""" SurveyNSEM
```

Comment:
Thanks for the docs! Would you mind moving this into the SurveyNSEM class? Then it will be added to the docs online